conformance-suites/2.0.0/conformance2/textures/misc/copy-texture-image.html

```diff
@@ -80,8 +80,12 @@
         "RG8",
         "RGB8",
         "RGBA8",
+        "R8_SNORM",
```

Comment:
I'm less sure about the omission of RGB  integer formats being deliberate-- they're not mentioned explicitly on the bug in the way that SNORM formats are. RGB integer formats aren't color renderable, but RGBA integer formats are, and it should be legal, I think, to copy from a source buffer created from, e.g., a RGBA8I texture, to a texel array using internalformat RGB8I, unless I'm forgetting something. CopyTexImage internalformat validation is a particularly hairy topic that the working group has revisited a number of times, so it is quite possible that the formats were missed.